spec/features/reviews_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+feature "recenzje" do
+
+  context "redaktor/recenzent, zgłoszenie, wersja w bazie danych" do
+    before do
+      Person.create!(name: "Andrzej", surname: "Kapusta", discipline: "filozofia", email: "a.kapusa@gmail.com", sex: "mężczyzna", roles: ['redaktor', 'recenzent'])
+      Submission.create!(person_id: Person.first, status: "odrzucony", polish_title: "Alicja w krainie czarów", english_title: "Alice 
+        in Wonderland", polish_abstract: "Słów parę o tej bajce", english_abstract: "Little about that story", polish_keywords: "alicja",
+        received: "19-01-2016", language: "polski")
+      ArticleRevision.create!(submission: Submission.first, version: "1", received: "19-01-2016", pages: "5")  
+    end
+    
+    context "2 recenzje w bazie danych" do
+      before do
+        Review.create!(article_revision: ArticleRevision.first, person: Person.first, status: "recenzja przyjęta", 
```

Comment:
Tutaj będzie trzeba podać `id: 2` - wynika to z istniejącego testu do recenzji. Zmienimy to później.